datapreview.tex

```diff
@@ -66,6 +66,7 @@ \subsection{Data Preview 1}
 
 In both DP1 scenarios, the exact processing pipelines and data products are still under discussion.
 At minimum,  DP1 will deliver visit-level calibrated images and catalogs to enable initial studies of observational and instrumental effects. 
+Attribution of known Solar System Objects is also expected.
```

Comment:
I think the wording is too strong  -- I would prefer to be clear that this is a stretch goal. For example, I would not want to delay the release of DP1 that contains visit-level calibrated images and catalogs in order to carry out atrribution of known SS Object (or any other data product).

Reply:
Any delay to the release of DP1 has a knock on impact on delivering DP2 and DR1

Reply:
If we're not able to associate SolSys at DP1 time, that will indicate /serious/ trouble. This is functionality we basically have now. I'm comfortable promising it.

Reply:
There are other data products that we would like to release, e.g. coadds, and if possible we will but I am not comfortable "promising" them.  We have a lot of functionality that work but is not yet tested on real LSST data, which is why I want to manage expectations conservatively for DP1.

Reply:
```suggestion
Additional data products, such as coadds or attribution of known Solar System Objects is a stretch goal for DP1"
```

Reply:
Proposal: by mid-October we will have demonstrated known-object association with ComCam (and should be running it routinely). The level of effort needed for SolSys-in-DP1 should also be well understood at that point (should be just a few extra columns in the Source table).

If we manage to do that, would it be possible to add this type of language into RTN-011 (before Nov 1st)?

Reply:
Thinking further .. aren't SSObjects identified as a part of DIA? We are not intending to make any DIA products available as part of DP1. DP1 is only single-epoch PVIs and catalogs of detections in those PVIs. There are no DIA products in DP1.

Reply:
We do not yet know what the DP1 dataset will look like. The ComCam on-sky program must support the engineering activities as its first priority with science-grade observing as a second priority.  We cannot make promises that depend on our ability to observe or the duration of observing during ComCam.

Reply:
Proposal:  In both DP1 scenarios, the exact processing pipelines and data products are still to be determined and depend strongly on the dataset that emerges from the commissioning on-sky observing program.
At minimum,  DP1 will deliver visit-level calibrated images and catalogs to enable initial studies of observational and instrumental effects.  Rubin will do its best to deliver the maximum of data products possible with the on-sky dataset, however at this time, all other data products described in the DPDD remain a stretch goal.

Reply:
I don't object to that wording.